datajoint/settings.py

```diff
@@ -7,6 +7,7 @@
 import pprint
 import logging
 import collections
+import collections.abc
```

Comment:
Nm, this has actually been handled correctly already. We can leave as is.

```suggestion
import collections
```